zerver/openapi/zulip.yaml

```diff
@@ -6204,6 +6204,11 @@ paths:
         **Note**: See [configuring message editing][config-message-editing]
         for detailed documentation on when users are allowed to edit topics.
 
+        **Changes**: Before Zulip 6.0 (feature level 150), editing
+        streams and topics of messages was forbidden if
+        `allow_message_editing` was `false`, regardless of the
+        `edit_topic_policy` or `move_messages_between_streams_policy`.
```

Comment:
So thinking about this change -- I believe it is removing entirely the ability to disable moving messages between streams/topics. I'm not sure we want to do that.

We could avoid that potential regression by first adding NOBODY to set the of options for `edit_topic_policy` and `move_message_between_streams_policy`. @alya thoughts?

Reply:
Sure, yeah, that sounds like a good approach to me.

Reply:
Yes, can do that and would be easy change. But for topic editing case the user would still be allowed to edit topics sent by themselves (and also others using different `propogate_mode` values) since `edit_topic_policy` is only for message sent by others. We want to change this also though such that all messages are treated same and `edit_topic_policy` affects all messages as mentioned in the issue. So, I think I can also add the commit for that change too (along with adding `Nobody` option) and then we can move forward with this.

But probably we first need to complete #22652 and then would be simpler to refactor the code for the above changes.